<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamTimeWidget</class>
 <widget class="QWidget" name="StreamTimeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="m_pBtnMvLeft">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前移</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditMove">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMvRight">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>后移</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSecMv">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelStart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始：</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelTimeStart">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelEnd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束：</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelTimeEnd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelStartSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMvLeftStart">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前移</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditMoveStart">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMvRightStart">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>后移</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSecMvStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMvLeftEnd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前移</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditMoveEnd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMvRightEnd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>后移</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSecMvEnd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelStartSet_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelStartSet_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>整体时间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
